src/pages/dev/amplifier/chain-integration/relay-messages.mdx

```diff
@@ -0,0 +1,157 @@
+# Relay messages with Amplifier
+
+import { Callout } from "/src/components/callout"
+
+Chains integrated with Amplifier can [pass GMP messages](https://docs.axelar.dev/dev/general-message-passing/overview) to one another through the following flow:
+
+1. When a user sends a message to the chain, a `callContract` event is emitted.
+1. The relayer picks up the event and calls [`verify_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L9) on the chain’s [Axelar (internal) gateway contract](https://github.com/axelarnetwork/axelar-amplifier/blob/main/doc/src/contracts/gateway.md).
+1. For chains that rely on verifier votes, the Axelar gateway calls [`verify_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L9) on the voting verifier, which prompts verifiers to begin voting.
+1. Once a threshold of successful votes have been cast for the message, an event is emitted.
+1. A relayer hears the event and calls [`route_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L14) on the chain’s Axelar gateway.
+1. The chain’s Axelar gateway passes the messages to the Amplifier router.
+1. The router passes the message to the destination chain’s Axelar gateway.
+1. The chain’s prover calls [`construct_proof`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/contracts/multisig-prover/src/msg.rs#L61), which starts the process of creating a signed batch that can be relayed back to the source chain. This method also passes the now-outbound message from the gateway to the prover.
+1. The prover starts a signing session with the multisig contract.
+1. Verifiers participate in the signing session.
+1. Once enough signatures have been submitted, the relayer gets the fully signed proof from the prover and relays the proof to the destination chain to approve the transactions.
+1. A relayer can now execute any approved transactions on the destination chain.
+
+## Axelar contract deployments
+
+- [`devnet-verifiers.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/9eb7341088cc0a2fd55e76377cd9b2a14f0989e9/axelar-chains-config/info/devnet-verifiers.json)
+- [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json)
+
+## Verify messages
+
+<Callout>
+This is temporarily permissioned to the admin contract established by the network.
```

Comment:
Not true. `update_verifier_set`  changes are temporarily permissioned, maybe these got confused?

I was wrong about this earlier.